src/lib/rest/rest.cpp

```diff
@@ -193,6 +193,21 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 
   LM_T(LmtUriParams, ("URI parameter:   %s: %s", key.c_str(), ciP->uriParam[key].c_str()));
 
+  //
+  // Now check the URI param has no invalid characters
+  // Except for the URI param 'q' that is not to be checked for invalid characters
+  //
+  if (key != "q")
```

Comment:
Not only in `q` parameter, also in the case of `value` in scopes of `type` SCOPE_TYPE_SINGLE_FILTER (or something like that, I don't remember the `#define` used right now...).